Source/Fuse.Controls.Native/SystemScreenConfigBase.uno

```diff
@@ -0,0 +1,123 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Fuse;
+using Fuse.Internal;
+
+namespace Fuse.Controls
+{
+	public class SystemScreenConfigBase : Behavior
```

Comment:
Please drop this pointless base-class (inline everything into `SystemScreenConfig`). Unneeded base-classes can't be removed from the API without breaking the API, and we generally don't use base-classes to share implementation details.